Dockerfile

```diff
@@ -9,7 +9,7 @@ COPY vendor/php-parser/composer.json /usr/src/app/vendor/php-parser/
 COPY vendor/php-parser/composer.lock /usr/src/app/vendor/php-parser/
 
 RUN curl --silent --location https://deb.nodesource.com/setup_5.x | bash -
-RUN apt-get update && apt-get install -y nodejs python openssh-client php5-cli php5-json
+RUN apt-get update && apt-get install -y nodejs npm python openssh-client php5-cli php5-json
```

Comment:
@ABaldwinHunter your issue is because of the dependency between the installation here and the nodesource install a line above. You should combine them to one `RUN` invocation with `&&` so that they are cached (and invalidated) together.